src/vecffe.c

```diff
@@ -738,8 +720,6 @@ static Obj FuncADD_ROWVECTOR_VECFFES_3(Obj self, Obj vecL, Obj vecR, Obj mult)
 **
 */
 
-static Obj MultVectorLeftOp; /* BH changed to static */
```

Comment:
Funny story: we actually failed to initialise `MultVectorLeftOp`, so the code path using it (deleted by this PR) would have crashed.